Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -506,6 +503,8 @@
 steps['RunZeroBias2021']={'INPUT':InputInfo(dataSet='/ZeroBias/Commissioning2021-v1/RAW',label='2021Commissioning',events=100000,location='STD', ls=Run2021Commissioning)}
 steps['RunHLTPhy2021']={'INPUT':InputInfo(dataSet='/HLTPhysics/Commissioning2021-v1/RAW',label='2021Commissioning',events=100000,location='STD', ls=Run2021Commissioning)}
 steps['RunNoBPTX2021']={'INPUT':InputInfo(dataSet='/NoBPTX/Commissioning2021-v1/RAW',label='2021Commissioning',events=100000,location='STD', ls=Run2021Commissioning)}
+steps['AlCaPhiSym2021']={'INPUT':InputInfo(dataSet='/AlCaPhiSym/Commissioning2021-v1/RAW',label='2021Commissioning',run=[346512],location='STD')}
+steps['AlCaLumiPixels2021']={'INPUT':InputInfo(dataSet='/AlCaLumiPixelsCountsExpress/Commissioning2021-v1/RAW',label='2021',run=[346512],events=100000,location='STD')}
```

Comment:
Why do you choose `label='2021'` instead of `label='2021Commissioning`' here?

Reply:
Hi @srimanob!
I didn't wrote this line actually, I just moved it down here from above to be together with the other 2021 datasets. Personally I don't have any preference on the label, do you prefer I change it to '2021Commissioning'?

Reply:
Hi @francescobrivio 
Ah, yeah. the line was somehow put above 2021 commissioning before. Would you mind to fix its label? Thanks.

Reply:
Done in 6e9efee